src/main/java/training/subsystems/wrist/Wrist.java

```diff
@@ -0,0 +1,54 @@
+package training.subsystems.wrist;
+
+import com.revrobotics.CANSparkBase;
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.math.geometry.Rotation2d;
+import utils.GBSubsystem;
+
+public class Wrist extends GBSubsystem {
+
+	private static CANSparkMax motor;
```

Comment:
final, not static
![image](https://github.com/user-attachments/assets/96298001-d61f-4a93-8a4b-6df250d19b7a)